content/plugins/commons-chunk-plugin.md

```diff
@@ -182,6 +187,23 @@ new webpack.optimize.CommonsChunkPlugin({
 })
 ```
 
+To add further control to an implicit common vendor chunk, you can additional logic before the `node_modules` context check to prevent
+specific modules from being included in the vendor chunk (such as stylesheets you want to keep in the original chunk):
```

Comment:
Sounds reasonable. I'd drop the rather obvious existing two lines and write something similar to "In order to obtain a single CSS file containing your application and vendor CSS, use the following `minChunks` function together with `ExtractTextPlugin`" (and add a link to the ExtractTextPlugin docs).